### DISCUSSION TITLE
Can I read series inputs from several series ports simultaneously?

### DISCUSSION THREAD

**Discussion by addy9908** [bonsai-rx/bonsai#2063] [Q&A]
I have 4 teensy boards that will send the series information out, and I plan to record them at the same time within one Bonsai code, by creating 4 series ports. Will it be possible to read these series input simultaneously without delay?
Sorry I do not have the teensy in hand yet, but just want to know the possibility before spending money.
Thanks.

**Comment by bruno-f-cruz** [comment]
You should be able to acquire from 4 independent serial ports w/o issues.

**Comment by addy9908** [comment]
Thanks Brunocruz.On Nov 27, 2024, at 03:13, brunocruz ***@***.***> wrote:﻿
Depends on what "exactly" exactly means :p, but in theory you should be able to acquire from 4 independent serial ports w/o issues.
